spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -73,32 +79,32 @@ interface UpgradeTimeout {
 - `timeoutHeight`: Timeout height indicates the height at which the counterparty must no longer proceed with the upgrade handshake. The chains will then preserve their original channel and the upgrade handshake is aborted.
 - `timeoutTimestamp`: Timeout timestamp indicates the time on the counterparty at which the counterparty must no longer proceed with the upgrade handshake. The chains will then preserve their original channel and the upgrade handshake is aborted.
 
-At least one of the timeoutHeight or timeoutTimestamp MUST be non-zero.
+At least one of the `timeoutHeight` or `timeoutTimestamp` MUST be non-zero.
 
 ### Store Paths
 
-#### Restore Channel Path
+#### Backup Channel Path
 
 The chain must store the previous channel end so that it may restore it if the upgrade handshake fails. This may be stored in the private store.
 
 ```typescript
-function restorePath(portIdentifier: Identifier, channelIdentifier: Identifier): Path {
-    return "channelUpgrade/ports/{portIdentifier}/channels/{channelIdentifier}/restore"
+function backupChannelPath(portIdentifier: Identifier, channelIdentifier: Identifier): Path {
+    return "channelEnds/ports/{portIdentifier}/channels/{channelIdentifier}/backup"
 }
 ```
 
-#### UpgradeError Path
+#### Upgrade Error Path
 
 The upgrade error path is a public path that can signal an error of the upgrade to the counterparty. It does not store anything in the successful case, but it will store a sentinel abort value in the case that a chain does not accept the proposed upgrade.
 
 ```typescript
-function errorPath(portIdentifier: Identifier, channelIdentifier: Identifier): Path {
-    return "channelUpgrade/ports/{portIdentifier}/channels/{channelIdentifier}/upgradeError"
+function upgradeErrorPath(portIdentifier: Identifier, channelIdentifier: Identifier): Path {
```

Comment:
We should prefix the channel path functions with `channel` as well since there are equivalent functions for connection which will cause a name collision